src/simdfunctionals/map.jl

```diff
@@ -266,6 +266,7 @@ end
 @inline _all_dense(t::Tuple{ArrayInterface.True,ArrayInterface.False,Vararg}) = false
 @inline _all_dense(t::Tuple{ArrayInterface.False,Vararg}) = false
 @inline all_dense() = true
+@inline all_dense(t::NTuple{N,T}) where {N,T} = true
```

Comment:
```suggestion
@inline all_dense(t::NTuple{N}) where {N} = true
```
should fix the test failures. An `NTuple` is already homogenous. If `N == 0`, then `T` would be an unbound arg.